README.md

```diff
@@ -18,7 +18,7 @@ Using REST API endpoints (requires WordPress 4.4+), or a Golang daemon, an event
 
 ## Installation ##
 
-1. Define `WP_CRON_CONTROL_SECRET` in `wp-config.php`, set to `false` to disable the REST API interface.
+1. Define `WP_CRON_CONTROL_SECRET` in `wp-config.php`
```

Comment:
I think we should leave this, helps guide the setup a bit better.

Reply:
It was regenerated by Grunt from readme.txt, I believe - probably need to update it there

Reply:
Ahh, right. I'll handle that.

Reply:
I already did that